docs/plugin_developers_guide/11_WSL2.rst

```diff
@@ -0,0 +1,328 @@
+Here are some useful features:
+
+- *Cut and paste text into and out of the Linux window*:
+  The Linux window behaves just like a Command Prompt window or a Powershell window.
+  You can copy text from any window and paste it into any other.
+- *Access the Linux file system from Windows*:
+  The Linux file system appears as a Network drive in Windows.
+  Open the Windows File Explorer and navigate to "*\\\\wsl$*."
+  You will see your Linux distributions appear as network folders.
+- *Access the Windows file system from Linux*:
+  From the *bash* command line, navigate to the mount point "*/mnt*."
+  You will see your Windows drive letters in this directory.
+  
+Preparing the Linux Distribution for Fledge
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The *systemd* service manager is not configured by default in an Ubuntu distribution running in WSL2.
+Since Fledge relies on *systemd*, you must run a script to enable it.
+From your home directory in the Ubuntu window, enter the commands:
+
+.. code-block:: bash
+
+   git clone https://github.com/DamionGans/ubuntu-wsl2-systemd-script.git
+   cd ubuntu-wsl2-systemd-script
+   bash ubuntu-wsl2-systemd-script.sh
+   
+Restart the Ubuntu distribution using *sudo reboot* or *sudo systemctl reboot*.
+When the distribution has restarted, run the command *systemctl*.
+You should see no error and a list of units.
+The script must be run *one time only*.
+Whenever you start up your Ubuntu distribution, *systemd* should be ready.
+
+Installing Fledge
+~~~~~~~~~~~~~~~~~
+
+Following the normal instructions for `Installing Fledge on Ubuntu <https://fledge-iot.readthedocs.io/en/latest/quick_start/installing.html#ubuntu-or-debian>`_.
+Make sure the package repository matches your version of Ubuntu.
+You can check the operating system version in your distribution with the command *hostnamectl* or *cat /etc/os-release*.
+
+Installing Visual Studio Code
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Navigate to the `Visual Studio Code <https://code.visualstudio.com>`_ webpage in your Windows browser.
+Click the *Download for Windows* button.
+Run the installer to install Visual Studio Code.
+
+Visual Studio Code is available for Microsoft Windows, Apple MacOS, and several Linux distributions.
+**Do not install the Linux build of Visual Studio Code in your Linux distribution in WSL2.**
+You will actually be launching Visual Studio Code for Windows from your Linux distribution!
+
+Starting the Linux Distribution
+-------------------------------
+
+Perform these steps every time you start your Linux distribution if you plan to run Fledge:
+
+Starting syslog
+~~~~~~~~~~~~~~~
+
+The system log */var/log/syslog* is not configured to run automatically in a Linux distribution in WSL2.
+Start *syslog* with the command:
+
+.. code-block:: bash
+
+   sudo service rsyslog start
+
+You must do this at every startup.
```

Comment:
You did it right to enable the system service `sudo systemctl enable SERVICE_NAME` and it will automatically boot up on startup. There may be some corrupted/wrong configuration of its unit file.